src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -780,4 +785,13 @@ def deleteNode(self, xpath):
 
         return True
 
+
```

Comment:
Hope you find no problem with the translation till now, by the way we don't support yang list inside another yang list. Which is needed for DSCP_TO_TC_MAP table.

Reply:
No issue found so far in translation. I will add more tables and tests as and when Yang models for other features are added.